packages/core/src/dialects/abstract/query-generator.js

```diff
@@ -1922,7 +1923,7 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
     const throughAttributes = through.attributes.map(attr => {
       let alias = `${externalThroughAs}.${Array.isArray(attr) ? attr[1] : attr}`;
 
-      if (this.options.minifyAliases) {
+      if (options.minifyAliases) {
```

Comment:
I haven't checked but do is `options` filled here everytime?

Reply:
Yes actually in `query-generator` we're only using `generateThroughJoin` at one place and I am actually passing the options bag like this:

```javascript
this.generateThroughJoin(include, includeAs, parentTableName.internalAs, topLevelInfo, { minifyAliases: options.minifyAliases });
```

So if done like above it should always be there even if undefined!

Reply:
Great, thanks for that info!